9000/alpine-jdk/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM java:8-jdk-alpine
+
+RUN apk --no-cache update && \
```

Comment:
Ok, sorry... no need for update here. `apk add --no-cache ...` will do the trick